File: Backend/WebMedicalApplication/BackendMedicalApplication/DTo/ReviewDto.cs
namespace BackendMedicalApplication.DTo
{
    public class ReviewDto
    {
        public int ReviewId { get; set; }
        public int DoctorId { get; set; }
        public int PatientId { get; set; }
        public int Rating { get; set; }
        public string Comment { get; set; }
        public DateTime CreatedAt { get; set; }
        public int HelpfulCount { get; set; } 
        public int NotHelpfulCount { get; set; }
    }
}

File: Frontend/FrontMedicalApplication/src/app/core/dtos/review.dto.ts
export interface ReviewDto {
  reviewId: number;
  doctorId: number;
  patientId: number;
  rating: number;
  comment: string;
  createdAt: Date;
  helpfulCount: number;
  notHelpfulCount: number;
  responseComment?: string;
  patientName?: string;
  doctorName?: string;
  voteType?: 'helpful' | 'not-helpful' | '';
}
